CMakeLists.txt

```diff
@@ -28,8 +28,7 @@ if (PROJECT_IS_TOP_LEVEL)
   if(BUILD_NVFUSER_BENCHMARK)
     include(cmake/Dependencies.cmake)
   endif()
-  set(NVFUSER_LIB_DIR "${NVFUSER_ROOT}/nvfuser/lib")
-  file(MAKE_DIRECTORY ${NVFUSER_LIB_DIR})
+  set(CMAKE_INSTALL_PREFIX "${NVFUSER_ROOT}/nvfuser")
```

Comment:
:clap: Would you mind double checking if we are dumping any junk here after standalone build?

Reply:
I don't see anything in the `nvfuser` folder.

Reply:
You mean nothing besides the actual library and python scripts? That's good enough then.

Reply:
Yes, nothing extra is added to the `nvfuser` folder.